alws/crud/test.py

```diff
@@ -16,22 +16,32 @@
 from alws.utils.file_utils import download_file
 from alws.utils.parsing import parse_tap_output, tap_set_status
 
+
 async def create_test_tasks_for_build_id(db: Session, build_id: int):
     async with db.begin():
-        ## We get all build_tasks with the same build_id
-        ## and whose status is COMPLETED
+        # We get all build_tasks with the same build_id
+        # and whose status is COMPLETED
         build_task_ids = (await db.execute(
             select(models.BuildTask.id).where(
                 models.BuildTask.build_id == build_id,
                 models.BuildTask.status == BuildTaskStatus.COMPLETED
             )
         )).scalars().all()
 
+        build = (await db.execute(select(models.Build).where(
+            models.Build.id == build_id).options(
+            selectinload(models.Build.repos)
+        ))).scalars().first()
+        test_log_repository = [i for i in build.repos
+                               if i.type == 'test_log'][0]
```

Comment:
we can fall here if `build.repos` is empty, we can use here `next` and `generator` instead